src/_guides/language/language-tour.md

```diff
@@ -720,6 +721,83 @@ information about lists, see [Generics](#generics) and
 [Collections](/guides/libraries/library-tour#collections).
 
 
+### Sets
+
+A set in Dart is an unordered collection of unique items.
+Dart support for sets is provided by set literals and the [Set][] type.
+
+<aside class="alert alert-info" markdown="1">
+  **Version note:** Although the Set _type_ has always been a core part of Dart,
+  set _literals_ were introduced in Dart 2.2.
+</aside>
+
+Here is a simple Dart set, created using a set literal:
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (set-literal)"?>
+{% prettify dart %}
+var halogens = {'fluorine', 'chlorine', 'bromine', 'iodine', 'astatine'};
+{% endprettify %}
+
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  The analyzer infers that `halogens` has the type
```

Comment:
Why only "the analyzer"?
Maybe "The Dart language " ...

Reply:
Changed to the more neutral 'It is inferred that...'

Reply:
To avoid the passive voice, I've changed this to "Dart infers" here. I've also updated the other two places where we had "The analyzer infers".

If we want to be more precise, we could say "Dart tools infer that..." but I'm not sure whether that's helpful.